lib/tasks/locale.rake

```diff
@@ -204,6 +204,32 @@ namespace :locale do
       end
     end
 
+    js_plugins = {
+      'ui-components' => {
+        'en'    => 'https://raw.githubusercontent.com/ManageIQ/ui-components/master/locale/en/ui-components.po',
+        'es'    => 'https://raw.githubusercontent.com/ManageIQ/ui-components/master/locale/es/ui-components.po',
+        'fr'    => 'https://raw.githubusercontent.com/ManageIQ/ui-components/master/locale/fr/ui-components.po',
+        'ja'    => 'https://raw.githubusercontent.com/ManageIQ/ui-components/master/locale/ja/ui-components.po',
+        'pt_BR' => 'https://raw.githubusercontent.com/ManageIQ/ui-components/master/locale/pt_BR/ui-components.po',
+        'zh_CN' => 'https://raw.githubusercontent.com/ManageIQ/ui-components/master/locale/zh_CN/ui-components.po',
+      }
+    }
+
+    plugins_dir = File.join(Rails.root, 'locale/plugins')
+    Dir.mkdir(plugins_dir, 0700)
+    js_plugins.each do |plugin, content|
+      plugin_dir = File.join(plugins_dir, plugin)
+      Dir.mkdir(plugin_dir)
+      content.each do |lang, url|
+        lang_dir = File.join(plugin_dir, lang)
+        Dir.mkdir(lang_dir)
+        lang_file = "#{lang_dir}/#{url.split('/')[-1]}"
+        system "curl -o #{lang_file} #{url}"
```

Comment:
This should probably use `system!` (from `ManageIQ::Environment`)

Right now, all the urls create a file saying: "404: Not Found"

Reply:
Fixed. And also added an `ensure` block at the end of the task (the file deletion part).